From 7483e1ad82099905e2f462be333074c8f8942f13 Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Tue, 22 Apr 2025 19:45:32 +0200
Subject: [PATCH 69/84] kdeb: fix deb build for R3+R4

due to the set +u unassigned variables will be handled as error
and so build fails.
---
 scripts/package/builddeb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 4c2142959768..11429129e044 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -63,6 +63,7 @@ install_linux_image () {
 	*)
 		source_image_path="./$board.itb"
 		installed_image_path=boot/$board-${KERNELRELEASE}.itb
+		DTBFILE=""
 		case $board in
 		bpi-r2)
 			mkdir -p "${pdir}/boot/bananapi/$board/linux/dtb"
-- 
2.30.2

